File: interview questions/moveElementToEnd.js
function moveElementToEnd(array, toMove) {
  let i = 0;

  for (let j = 1; j < array.length; j++) {
    let left = array[i];
    let right = array[j];

    if (left !== right && right !== toMove) {
      [array[i], array[j]] = [array[j], array[i]];
      i++;
    }
  }

  return array;
}

File: interview questions/moveElementToEnd.py
def moveElementToEnd(array, toMove):
    i = 0
	
	for j in range(1, len(array)):
		if array[i] != array[j] and array[j] != toMove:
			array[i], array[j] = array[j], array[i]
			i += 1
	
	return array